src/arch/riscv/pagetable_walker.cc

```diff
@@ -403,6 +406,15 @@ Walker::WalkerState::stepWalk(PacketPtr &write)
                         if (!pte.d && mode != BaseMMU::Write)
                             entry.pte.w = 0;
                         doTLBInsert = true;
+
+                        // Update statistics for completed page walks
+                        if (level == 1)
+                            walker->pagewalkerstats.num_2mb_walks++;
+                        else if (level == 0)
+                            walker->pagewalkerstats.num_4kb_walks++;
```

Comment:
I would *strongly* prefer `{ }` curly braces around this. Especially without an `else`

Reply:
Thanks!